src/django/contricleaner/lib/serializers/row_serializers/row_composite_serializer.py

```diff
@@ -17,6 +20,7 @@ def __init__(self):
             RowSectorSerializer(),
             RowCountrySerializer(),
             RowEmptySerializer(),
+            RowRequiredFieldsSerializer(),
```

Comment:
It should go after RowEmptySerializer which is simply add all feeds to the result